src/components/Navbar/Navbar.jsx

```diff
@@ -39,14 +41,33 @@ function Navbar() {
     try {
       await logoutUser().unwrap();
       dispatch(logout());
-      navigate(ROUTER_PATHS.HOME);
+      setModalData({
+        type: MODAL_TYPES.SUCCESS,
+        title: MODAL_TYPES.LOGOUT_SUCCESS,
+        message: MODAL_TYPES.LOGOUT_SUCCESS_MESSAGE,
+      });
+
+      setTimeout(() => {
+        setModalData(null);
+        navigate(ROUTER_PATHS.HOME);
+      }, 2000);
+
```

Comment:
Save `2000` into a variable. 
Also you need to clean the timout if no longer used (Inside use effect on the return function)